service/gcs/runtime/runc/runc_test.go

```diff
@@ -182,7 +174,7 @@ var _ = Describe("runC", func() {
 			c  runtime.Container
 		)
 		JustBeforeEach(func() {
-			c, err = rtime.CreateContainer(id, bundle, createAllStdioOptions)
+			c, err = rtime.CreateContainer(id, bundle, &stdio.ConnectionSet{})
```

Comment:
Oh, yeah, there are no tests which explicitly validate it. It would still be good to make sure the code gets exercised, though, so at the very least we'll be able to catch any simple errors.